README.md

```diff
@@ -1 +1,63 @@
-# ironfoudation-team-project
+# WOOF application 
+
+## Table of contents
+* [General info](#general-info)
+* [Technologies](#technologies)
+* [Installation](#installation)
+* [Authors](#authors)
+* [Notes](#notes)
+
+## General info
+
+This is the application for iOS.
+Woof application connects dog owners with pet sitters which contributes to build trusted relationships between them. The app makes it easy for nearby pet owners to find pet sitting and makes convienience of responding to jobs for pet sitters from phone.
+App also makes managing sitting bussines and pet owner's schedule. They can choose their own schedule, prices, and requirements for servise. 
+
+**Features include**
+- Secure store all of pet owners' and pet sitters' data base. 
```

Comment:
typo:
```suggestion
- Secure store all of pet owners and pet sitters databases. 
```

Reply:
I wrote in singular form, using possessive syntax. But you are right. It looks now much better. 
Thanks, fixed it.